lib/rubocop/config_obsoletion.rb

```diff
@@ -45,19 +45,23 @@ def reject_obsolete!
 
     private
 
+    LOAD_RULES_CACHE = {} # rubocop:disable Style/MutableConstant
+    private_constant :LOAD_RULES_CACHE
```

Comment:
In this repository, `private_constant`s are implicitly written at the top of class declaration. Can you please move the constant and `private_constant` to be near `PARAMETER_RULE_CLASSES` constant (L14).

Reply:
Moved.